addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -492,8 +492,7 @@ def test_initial_data(self):
     @users('user_sales_manager')
     def test_lead_convert_batch_internals(self):
         """ Test internals of convert wizard, working in batch mode """
-        date = Datetime.from_string('2020-01-20 16:00:00')
-        self.crm_lead_dt_mock.now.return_value = date
+        date = self.env.cr.now()
```

Comment:
Note that we could check date_open and date_conversion values prior to calling ``action_apply``, just to be sure it is updated by the method itself.